django/utils/timesince.py

```diff
@@ -66,26 +71,34 @@ def timesince(d, now=None, reversed=False, time_strings=None):
     delta -= datetime.timedelta(leapdays)
 
     # ignore microseconds
-    since = delta.days * 24 * 60 * 60 + delta.seconds
-    if since <= 0:
+    total_seconds_since = delta.days * 24 * 60 * 60 + delta.seconds
+    if total_seconds_since <= 0:
```

Comment:
Changing the names of variables which is not necessary for this patch makes it harder to review, can we revert them? e.g. `total_seconds_since -> since`, or 
```
for index, (seconds_per_chunk, chunk_name) in enumerate(TIMESINCE_CHUNKS):
```
->
```
for i, (seconds, name) in enumerate(TIMESINCE_CHUNKS):
```